ext/gosu/gosu.i

```diff
@@ -1014,6 +1014,12 @@ namespace Gosu
     {
         $self->Gosu::Window::close();
     }
+
+    Gosu::Image to_image()
+    {
+        Gosu::Image image($self->to_bitmap());
```

Comment:
I don't think the temporary variable adds a lot here, better to just return Gosu::Image(...)